### ISSUE TITLE
servo.wait() method not working

### ISSUE THREAD

**Issue by fkws-k** [netlabtoolkit/VarSpeedServo#18]
Here is a simple example of this issue:

    #include <VarSpeedServo.h>
    VarSpeedServo servo;
    
    void setup() {
      servo.attach(10);
      servo.write(90);
      servo.wait();
      servo.detach();
    
    }
    void loop(){}

In this example, the servo doesn't move at all because it is detached before it can move despite the method `wait()` being called before `detach()`. I want to detach my servos after movement to keep them from jittering. Is there any way I can do that?

**Comment by pvanallen** [comment]
You could use the isMoving() function in your loop() to wait for the move to complete before you detach. Keep in mind that this library is designed to work primarily in the loop() because each time through the loop the servos are updated a little bit until the moves are complete.
